src/components/EventDetailsComponents/components/General.tsx

```diff
@@ -317,6 +357,11 @@ const General: React.FC<Props> = ({
   useEffect(() => {
     generalMountedRef.current = true;
   }, []);
+
+  useEffect(() => {
+      updateContinueWatching();
```

Comment:
can move the app to error if any user quickly closes this screen. change state of the already